fluttercommunity/flutter_workmanager
# [flutter_workmanager] Give support for Flutter null-safety

- Updates all versions to null-safety beta support.
- Converts dart code to give support for null-safety.
- Changed test-cases to support null-safety.

Testing instructions:

Add the following to your pubspec.yaml:

```
dependency_overrides:
  workmanager:
    git:
      url: https://github.com/parthdave93/flutter_workmanager.git
      ref: parthdave93/null_safety_migration
```

Pre-launch Checklist

  The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
  I read the Contributor Guide and followed the process outlined there for submitting PRs.
  I read the Tree Hygiene wiki page, which explains my responsibilities.
  I read and followed the Flutter Style Guide and the C++, Objective-C, Java-style guides.
  I added new tests to check the change I am making or the feature I am adding, or Hixie said the PR is tested exempt.
  I updated pubspec.yaml with an appropriate new version according to the pub versioning philosophy.
  I updated CHANGELOG.md to add a description of the change.
  I updated/added relevant documentation (doc comments with ///).
  All existing and new tests are passing.
